src/_pytest/terminal.py

```diff
@@ -1230,3 +1233,12 @@ def format_session_duration(seconds: float) -> str:
     else:
         dt = datetime.timedelta(seconds=int(seconds))
         return "{:.2f}s ({})".format(seconds, dt)
+
+
+def _raw_skip_reason(rep):
+    assert rep.skipped
+    assert len(rep.longrepr) == 3
+    _, _, reason = rep.longrepr
+    if reason.startswith("Skipped: "):
+        reason = reason[9:]
```

Comment:
```suggestion
        reason = reason[len("Skipped: "):]
```

Just for clarity. :+1:

(Probably worthwhile moving `"Skipped: "` to a variable and reuse in both places).